llrt_modules/src/modules/events/event.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use std::marker::PhantomData;
+
+use rquickjs::{prelude::Opt, Result, Value};
+
+use llrt_utils::object::ObjectExt;
+
+#[rquickjs::class]
+#[derive(rquickjs::class::Trace)]
+pub struct Event<'js> {
+    event_type: String,
+    bubbles: bool,
+    cancelable: bool,
+    composed: bool,
+    #[qjs(skip_trace)]
+    marker: PhantomData<&'js ()>,
```

Comment:
Why do we need the lifetime here?

Reply:
It was completely unnecessary at this time and has been removed.